pom.xml

```diff
@@ -117,6 +117,200 @@
 
         <!-- Version properties for this repository live in this repository -->
         <org.rrd4j.version>3.3.1</org.rrd4j.version>
+
+        <!--Frontend properties-->
+        <backbone.modelbinder.version>1.1.0</backbone.modelbinder.version>
+        <boon.version>0.34</boon.version>
+        <bouncy.version>1.60</bouncy.version>
+        <dom4j.bundle.version>1.6.1_5</dom4j.bundle.version>
+        <google-http-client.version>1.22.0</google-http-client.version>
+        <html5shiv.version>3.7.2</html5shiv.version>
+        <httpclient.version>4.5.6</httpclient.version>
+        <httpcore.version>4.4.10</httpcore.version>
+        <httpmime.version>${httpclient.version}</httpmime.version>
+        <iframe-resizer.version>2.6.2</iframe-resizer.version>
+        <jknack.handlebars.version>2.0.0</jknack.handlebars.version>
+        <jruby.version>9.0.4.0</jruby.version>
+        <marionette.version>2.4.5</marionette.version>
+        <picocontainer.version>1.3_1</picocontainer.version>
+        <underscore.version>1.8.3</underscore.version>
+        <yarn.version>v1.5.1</yarn.version>
+
+        <!--Test properties-->
+        <hamcrest-all.servicemix.version>1.3_1</hamcrest-all.servicemix.version>
+        <hamcrest-all.version>1.3</hamcrest-all.version>
+        <hamcrest-junit.version>2.0.0.0</hamcrest-junit.version>
+        <hamcrest-optional.version>1.0</hamcrest-optional.version>
+        <junit.version>4.12</junit.version>
+        <mockito-core.version>2.8.47</mockito-core.version>
+        <mockito-all.version>1.10.19</mockito-all.version>
```

Comment:
❓ Why pull in mockito to DDF but leave groovy in the parent? Mostly curious why only some of the test modules are getting moved

Reply:
groovy is pulled in via codice-test along with spock

Reply:
Ahhh okay

Reply:
Any reason mockito isn't pulled in by codice-test 🤔

Reply:
because codice-test is not a repo of versions. It is a repo of test utilities where we have a `spock-all` artifact designed to combine in one artifact all spock required dependencies. This is what replaced the old `spock-shaded`. As such, we pull these dependencies as a pom artifact now.